maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/controller/CCOutcomeController.java

```diff
@@ -87,17 +84,19 @@ public ResponseEntity<RepOrderCCOutcomeDTO> create(@Parameter(description = "Rep
                     schema = @Schema(implementation = ErrorDTO.class)
             )
     )
-    public ResponseEntity<RepOrderCCOutcomeDTO> update(@Parameter(description = "RepOrder CC outcome data",
-            content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
-                    schema = @Schema(implementation = RepOrderCCOutcome.class)
-            )
-    ) @RequestBody RepOrderCCOutcome repOrderCCOutCome) {
+    public ResponseEntity<RepOrderCCOutcomeDTO> update(
+            @Parameter(description = "RepOrder CC outcome data",
+                    content = @Content(mediaType = MediaType.APPLICATION_JSON_VALUE,
+                            schema = @Schema(implementation = RepOrderCCOutcome.class)
```

Comment:
Similar indentation comments as above

Reply:
For my future self, same comments as previous around the `create` method in the CCOutcomeController.